### PR TITLE
Add language support for Cap’n Proto format

### PR DESCRIPTION
**Pull request by angelodlfrtr**
Tree-sitter grammar and queries from https://github.com/amaanq/tree-sitter-capnp.

### CODE REVIEW

File: runtime/queries/capnp/highlights.scm [hunk 1]
```diff
@@ -0,0 +1,141 @@
+; Preproc
+
+(unique_id) @preproc
```
Comment (the-mikedavis) [inline runtime/queries/capnp/highlights.scm:3]: It looks like these queries use neovim captures. We use a slightly different set of captures documented here: https://docs.helix-editor.com/master/themes.html#syntax-highlighting

For example, this should be `@keyword.directive`, `@include` should be `@keyword.control.include`, and `@method` should be `@function.method`

File: runtime/queries/capnp/injections.scm [hunk 2]
```diff
@@ -0,0 +1 @@
+(comment) @comment
```
Comment (the-mikedavis) [inline runtime/queries/capnp/injections.scm:1]:
```suggestion
((comment) @injection.content
 (#set! injection.language "comment"))
```

File: runtime/queries/capnp/locals.scm
Comment (the-mikedavis) [inline runtime/queries/capnp/locals.scm:1]: For locals we use the captures `@local.scope`, `@local.definition` and `@local.reference`

File: runtime/queries/capnp/highlights.scm (hunk 1 above)
Comment (angelodlfrtr) [reply runtime/queries/capnp/highlights.scm]: Updated in [ac50eb3](https://github.com/helix-editor/helix/pull/6325/commits/ac50eb3fd5ead297a5ef5c277f99b4057ed927be)

Comment (angelodlfrtr) [reply runtime/queries/capnp/highlights.scm]: Thanks for the review! The result is much better with the correct set of captures.

File: runtime/queries/capnp/indents.scm [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  (struct)
+  (struct_shorthand)
+  (union)
+] @indent
+
+((struct_shorthand (property)) @aligned_indent
+  (#set! "delimiter" "()"))
+
+((const_list (const_value)) @aligned_indent
+  (#set! "delimiter" "[]"))
```
Comment (pickfire) [inline runtime/queries/capnp/indents.scm:20]: I don't think we have `@aligned_indent`?

Comment (angelodlfrtr) [reply runtime/queries/capnp/indents.scm]: Thanks @pickfire 
From https://docs.helix-editor.com/guides/indent.html#adding-indent-queries, no.
I replaced `@indent_end` by `@outer`, removed `@branch` and `@aligned_indent`.
Only "scope" property is supported from https://github.com/helix-editor/helix/blob/master/helix-core/src/indent.rs#L433

**Comment by angelodlfrtr** [comment]
Hi @pickfire @the-mikedavis, thanks for your reviews ! Does the changes seems good to you ?

**Review by the-mikedavis — APPROVED** [review]

**Pull request merged**